frontend/src/app/app.component.ts

```diff
@@ -1,62 +1,67 @@
-import { Component, OnInit } from '@angular/core';
-import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
-import { PatientInfo } from './model/PatientInfo';
-import { InsuranceCompany } from './model/InsuranceCompany';
-import { MatDialog } from '@angular/material/dialog';
-import { DialogComponent } from './components/dialog/dialog.component';
-import { PatientService } from './services/patient.service';
+import {Component, OnInit} from '@angular/core';
+import {FormBuilder, FormGroup, Validators} from '@angular/forms';
+import {PatientInfo} from './model/PatientInfo';
+import {InsuranceCompany} from './model/InsuranceCompany';
+import {MatDialog} from '@angular/material/dialog';
+import {DialogComponent} from './components/dialog/dialog.component';
+import {PatientService} from './services/patient.service';
+import {validatePersonalNumber, validatePhoneNumber} from './app.validators';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent implements OnInit {
+
+  constructor(
+    private formBuilder: FormBuilder,
```

Comment:
mozna bych tam nechala ty podtrzitka u privatnich veci, akorat tam musim pridat stejny linter jak na TXM. Ale nech to tak, opravim to pak

Reply:
tak na private mas slovo `private` :)